test/controlplane/services/nodeport/v1.24/state1.yaml

```diff
@@ -293,8 +293,8 @@ items:
       type: PodScheduled
     containerStatuses:
     - containerID: containerd://cc84c3c87f6ccacbe724ee0fad740dd250e65b224273bcb9591927a6d0ed6987
-      image: docker.io/cilium/json-mock:1.2
-      imageID: docker.io/cilium/json-mock@sha256:941e03da57551dd4a71f351b35650c152a1192ac1df717e43ee58b5aa2b8e241
+      image: quay.io/cilium/json-mock:v1.3.3@sha256:f26044a2b8085fcaa8146b6b8bb73556134d7ec3d5782c6a04a058c945924ca0
+      imageID: quay.io/cilium/json-mock:v1.3.3@sha256:f26044a2b8085fcaa8146b6b8bb73556134d7ec3d5782c6a04a058c945924ca0
```

Comment:
What's `imageID` here? Do we need to keep it if it's identical to `image`?

Reply:
good point, this is just a dump from existing kind cluster as part of control plane tests, I don't think it will be used any more, but I will keep the same format (i.e. no version id, but only sha)